Products/ZenModel/Device.py

```diff
@@ -1124,7 +1124,7 @@ def updateDevice(self,**kwargs):
 
         # apply any zProperties to self
         for prop, value in zProperties.items():
-            if value is not None:
+            if value is not None and value != "":
```

Comment:
The value of `value` can also be a boolean and `False` is a legitimate value for a zProperty.  The fix for ZEN-16601 is to support non-string values and the change above is to exclude two specific values that aren't generally useful for zProperties.